sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/AbstractProjectConfigurationTest.java

```diff
@@ -98,35 +98,36 @@ public void noWarningsWhenGettingProtobufPathAndNoPropertyAvailable_TestProject(
   public void giveWarningsWhenGettingProtobufPathAndNoFolderAvailable() throws IOException {
     Path path1 = createProtobufOut("report");
 
-    Configuration configuration = createEmptyConfiguration();
+    Configuration configuration = createEmptyMockConfiguration();
     when(configuration.getStringArray("sonar.cs.analyzer.projectOutPaths")).thenReturn(new String[] {path1.toString(), "non-existing"});
 
     AbstractProjectConfiguration config = createAbstractProjectConfiguration(configuration);
     assertThat(config.protobufReportPaths()).containsOnly(path1.resolve("output-cs"));
-    assertThat(logTester.logs(LoggerLevel.WARN)).hasSize(1);
-    assertThat(logTester.logs(LoggerLevel.WARN).get(0)).matches(s -> s.startsWith("Analyzer working directory does not exist"));
+    assertThat(logTester.logs(LoggerLevel.WARN)).hasOnlyOneElementSatisfying(s -> s.startsWith("Analyzer working directory does not exist"));
   }
 
   @Test
   public void giveWarningsWhenGettingOldProtobufPathAndNoFolderAvailable() {
-    Configuration configuration = createEmptyConfiguration();
+    Configuration configuration = createEmptyMockConfiguration();
     when(configuration.get("sonar.cs.analyzer.projectOutPath")).thenReturn(Optional.of("non-existing"));
 
     AbstractProjectConfiguration config = createAbstractProjectConfiguration(configuration);
     assertThat(config.protobufReportPaths()).isEmpty();
-    assertThat(logTester.logs(LoggerLevel.WARN)).hasSize(1);
-    assertThat(logTester.logs(LoggerLevel.WARN).get(0)).matches(s -> s.startsWith("Analyzer working directory does not exist"));
+    assertThat(logTester.logs(LoggerLevel.WARN)).hasOnlyOneElementSatisfying(s -> s.startsWith("Analyzer working directory does not exist"));
   }
 
   @Test
   public void informHowManyProtoFilesAreFound() throws IOException {
-    Configuration configuration = createEmptyConfiguration();
+    Configuration configuration = createEmptyMockConfiguration();
     mockProtobufOutPaths(configuration);
 
     AbstractProjectConfiguration config = createAbstractProjectConfiguration(configuration);
     assertThat(config.protobufReportPaths()).isNotEmpty();
-    assertThat(logTester.logs(LoggerLevel.DEBUG)).hasSize(2);
-    assertThat(logTester.logs(LoggerLevel.WARN)).allMatch(s -> s.endsWith("contains no .pb file(s). No protobuf files will be loaded from this directory."));
```

Comment:
lol, talk about test code quality :)) 

indeed, as it was empty, all were matching